experimental/traffic-portal/nightwatch/globals/globals.ts

```diff
@@ -54,9 +54,12 @@ import {
 	RequestCacheGroup,
 	ResponseCacheGroup,
 	ResponsePhysicalLocation,
-	RequestPhysicalLocation
+	RequestPhysicalLocation, RequestType
```

Comment:
this should be on its own line, in keeping with the way the rest of this statement is organized